jdaviz/app.py

```diff
@@ -2031,13 +2031,21 @@ def vue_data_item_remove(self, event):
         data = self.data_collection[data_label]
         orientation_plugin = self._jdaviz_helper.plugins.get("Orientation")
         if orientation_plugin is not None:
+            from jdaviz.configs.imviz.helper import base_wcs_layer_label
             orient = orientation_plugin.orientation.selected
+            if orient == data_label:
+                orient = base_wcs_layer_label
             self._reparent_subsets(data, new_parent=orient)
         else:
             self._reparent_subsets(data)
 
-        # Make sure the data isn't loaded in any viewers
+        # Make sure the data isn't loaded in any viewers and isn't the selected orientation
         for viewer_id in self._viewer_store:
+            if orientation_plugin is not None:
+                orientation_plugin.viewer.selected = viewer_id
```

Comment:
Won't user see the viewer selection change every iteration when L2045 is run?

Reply:
Yes...I might be able to do this more elegantly without changing things in the plugin 🤔

Reply:
Your comment made me realize I didn't need to touch the plugin at all for the fix for bug 3, updated to a much better fix. Getting a fresh pair of eyes on things helps so much sometimes.